src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt

```diff
@@ -121,6 +121,7 @@ fun getReadOnlyClickableHtmlJEditorPane(
             BrowserHyperlinkListener.INSTANCE.hyperlinkUpdate(it)
         }
         if (noBorder) border = null
+        name = htmlText
```

Comment:
Why set the htmlText as name?

Reply:
for tests
easiest way to distinguish different JEditorPane from each other